perfmetrics/scripts/presubmit_test/pr_perf_test/build.sh

```diff
@@ -15,55 +17,83 @@ set -e
 wget -O go_tar.tar.gz https://go.dev/dl/go1.20.5.linux-amd64.tar.gz
 sudo rm -rf /usr/local/go && tar -xzf go_tar.tar.gz && sudo mv go /usr/local
 export PATH=$PATH:/usr/local/go/bin
-echo Installing fio
-sudo apt-get install fio -y
 
 # Run on master branch
 cd "${KOKORO_ARTIFACTS_DIR}/github/gcsfuse"
-git checkout master
-echo Mounting gcs bucket for master branch
-mkdir -p gcs
-GCSFUSE_FLAGS="--implicit-dirs --max-conns-per-host 100"
-BUCKET_NAME=presubmit-perf-tests
-MOUNT_POINT=gcs
-# The VM will itself exit if the gcsfuse mount fails.
-CGO_ENABLED=0 go run . $GCSFUSE_FLAGS $BUCKET_NAME $MOUNT_POINT
-touch result.txt
-# Running FIO test
-chmod +x perfmetrics/scripts/presubmit/run_load_test_on_presubmit.sh
-./perfmetrics/scripts/presubmit/run_load_test_on_presubmit.sh
-sudo umount gcs
-
 # Fetch PR branch
 echo '[remote "origin"]
          fetch = +refs/pull/*/head:refs/remotes/origin/pr/*' >> .git/config
 git fetch origin
-echo checkout PR branch
-git checkout pr/$KOKORO_GITHUB_PULL_REQUEST_NUMBER
 
-# Executing integration tests
-GODEBUG=asyncpreemptoff=1 CGO_ENABLED=0 go test ./tools/integration_tests/... -p 1 --integrationTest -v --testbucket=gcsfuse-integration-test -timeout 15m
+# execute perf tests.
+if [[ "$perfTestStr" == *"execute-perf-test"* ]];
+then
+  # Installing requirements
+  echo Installing python3-pip
+  sudo apt-get -y install python3-pip
+  echo Installing libraries to run python script
+  pip install google-cloud
+  pip install google-cloud-vision
+  pip install google-api-python-client
+  pip install prettytable
+  echo Installing fio
+  sudo apt-get install fio -y
+
+  # Executing perf tests for master branch
+  git checkout master
+  echo Mounting gcs bucket for master branch
+  mkdir -p gcs
+  GCSFUSE_FLAGS="--implicit-dirs --max-conns-per-host 100"
+  BUCKET_NAME=presubmit-perf-tests
+  MOUNT_POINT=gcs
+  # The VM will itself exit if the gcsfuse mount fails.
+  CGO_ENABLED=0 go run . $GCSFUSE_FLAGS $BUCKET_NAME $MOUNT_POINT
+  touch result.txt
+  # Running FIO test
+  chmod +x perfmetrics/scripts/presubmit/run_load_test_on_presubmit.sh
+  ./perfmetrics/scripts/presubmit/run_load_test_on_presubmit.sh
+  sudo umount gcs
+
+  # Executing perf tests for PR branch
+  echo checkout PR branch
+  git checkout pr/$KOKORO_GITHUB_PULL_REQUEST_NUMBER
+  echo Mounting gcs bucket from pr branch
+  mkdir -p gcs
+  # The VM will itself exit if the gcsfuse mount fails.
+  CGO_ENABLED=0 go run . $GCSFUSE_FLAGS $BUCKET_NAME $MOUNT_POINT
+  # Running FIO test
+  chmod +x perfmetrics/scripts/presubmit/run_load_test_on_presubmit.sh
+  ./perfmetrics/scripts/presubmit/run_load_test_on_presubmit.sh
+  sudo umount gcs
+
+  # Show results
+  echo showing results...
+  python3 ./perfmetrics/scripts/presubmit/print_results.py
+fi
+
+# Execute integration tests.
+if [[ "$integrationTestsStr" == *"execute-integration-tests"* ]];
+then
+  echo checkout PR branch
+  git checkout pr/$KOKORO_GITHUB_PULL_REQUEST_NUMBER
```

Comment:
Here, also we can use quit mode. This will remove extra logs - https://screenshot.googleplex.com/3cLmHMtPPtYpVHq

Ref: https://stackoverflow.com/questions/8943693/can-git-operate-in-silent-mode

Reply:
This two get removed. You refered these two in the bug.

https://screenshot.googleplex.com/8hBwWVFj43gGWXi
https://screenshot.googleplex.com/AYxjQvvDoxmdUFv

Reply:
Fetch origin is generating logs, that was resolved